### PR TITLE
Select Food Function

### PR DESCRIPTION
**Pull request by SHAUNGOH97**
-Select food adjusts the budget
-> Selected food is currently not saved
-> Not able to remove a selected food
-Some changes in foodcard UI

### CODE REVIEW

**Review by SHAUNGOH97 — COMMENTED** [review]
Fixed checkstyles

**Review by SHAUNGOH97 — COMMENTED** [review]
Fixing Checkstyle error

**Review by SHAUNGOH97 — COMMENTED** [review]
Fixing checkstyles

File: src/main/java/seedu/foodiebot/logic/commands/SelectItemCommand.java [hunk 1]
```diff
@@ -2,19 +2,106 @@
...
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD
+            + "Parameters: "
+            + "FOOD_INDEX \n"
+            + "Example: "
+            + COMMAND_WORD
+            + " "
+            + "1 ";;
+    public static final String MESSAGE_SUCCESS = "You have selected: %s, this costs: $%.2f\n";
+    public static final String MESSAGE_SUCCESS_BUDGET = "You have selected: %s, this costs: $%.2f\n"
```
Comment (lharland) [inline src/main/java/seedu/foodiebot/logic/commands/SelectItemCommand.java:34]: select item uses the message from budget?

Comment (SHAUNGOH97) [reply src/main/java/seedu/foodiebot/logic/commands/SelectItemCommand.java:34]: Oh I meant, if a budget is set, it uses that success message.
If no budget is set, then it uses the regular message success

**Pull request merged**